### PR TITLE
ez_http_tag_location → ez_http_cache_tag_location

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | Ibexa DXP (eZ Platform v3), ezsystems/ezplatform-http-cache v2.1.3+

Replace `ez_http_tag_location` ([kept for 2.5 BC](https://github.com/ezsystems/ezplatform-http-cache/commit/81dab5a92faa06f2c0068aee65e5d0383db7a12f)) with `ez_http_cache_tag_location`.

### CODE REVIEW

**Review by adamwojs — APPROVED** [review]
@ezsystems/documentation-team Could you please add `ez_http_cache_tag_location` and other HTTP cache related function to https://doc.ibexa.co/en/latest/guide/twig_functions_reference/ ?

File: docs/guide/http_cache.md [hunk 1]
```diff
@@ -812,12 +812,12 @@ $response->headers->set('X-Location-Id', '123,212,42');
...
+1\. `ez_http_cache_tag_location()`
 
 For full content tagging when inline rendering, use the following:
 
 ``` html+twig
-{{ ez_http_tag_location(location) }}
+{{ ez_http_cache_tag_location(location) }}
 ```
 
 2\. `ez_http_tag_relation_ids()` or `ez_http_tag_relation_location_ids()`
```
Comment (vidarl) [inline docs/guide/http_cache.md:823]: Likely, these functions should also have the `ez_http_cache` prefix

**Review by vidarl — CHANGES_REQUESTED** [review]

**Review by dabrt — APPROVED** [review]

File: docs/guide/http_cache.md (hunk 1 above)
Comment (dabrt) [reply docs/guide/http_cache.md:823]: @adamwojs 
Could you please confirm that 
`ez_http_tag_relation_ids()` and `ez_http_tag_relation_location_ids()`
should be updated in the docs to 
`ez_http_cache_tag_relation_ids()` and `ez_http_cache_tag_relation_location_ids()`

Comment (adriendupuis) [reply docs/guide/http_cache.md:823]: @vidarl `ez_http_cache_tag_location` doesn't exist (yet).And the goal of https://github.com/ezsystems/ezplatform-http-cache/pull/143 wasn't to create it. Could it be done in another PR?

Comment (adriendupuis) [reply docs/guide/http_cache.md]: @vidarl 

`ez_http_cache_tag_relation_ids` and `ez_http_cache_tag_relation_location_ids`  added in https://github.com/ezsystems/ezplatform-http-cache/pull/143

Doc updated in 4f9b4104442523425640b11a78acf602f611ce6d to have this two new Twig functions with the `ez_http_cache_tag_` prefix.

**Review by vidarl — COMMENTED** [review]
Should we still list the function names that is deprecated? so that people searching for it still finds it?

Not sure how the policy is for deprecated things, honestly @dabrt

**Review by adamwojs — APPROVED** [review]

**Comment by dabrt** [comment]
> Not sure how the policy is for deprecated things, honestly @dabrt

We do not mention them in the docs, the policy is to list deprecations, like here:
https://doc.ibexa.co/en/master/releases/ez_platform_v3.0_deprecations/#functions-renamed

For whatever reason these deprecations have not been reported at v3.0 and this branch has already been abandoned. At this point we should simply replace the deprecated signatures with proper ones.

**Comment by dabrt** [comment]
Hi @adriendupuis ,
Is there any traction in this PR? Are we waiting for further approvals?

**Comment by adriendupuis** [comment]
> Hi @adriendupuis ,
> Is there any traction in this PR? Are we waiting for further approvals?

Hello @dabrt
There was some discussions on the code part about deprecating more than this function. I have some rebasing to do regarding since which version it should have been deprecated. I didn't find the time to finish this yet.

**Comment by adriendupuis** [comment]
Those Twig function names have been updated in #1403

**Pull request closed without merging**